d2renderers/d2svg/d2svg.go

```diff
@@ -1838,18 +1839,21 @@ func Render(diagram *d2target.Diagram, opts *RenderOpts) ([]byte, error) {
 	if opts.Center {
 		alignment = "xMidYMid"
 	}
-	fitToScreenWrapper := fmt.Sprintf(`<svg %s d2Version="%s" preserveAspectRatio="%s meet" viewBox="0 0 %d %d"%s>`,
+	fitScreenWrapper := fmt.Sprintf(`<svg %s d2Version="%s" preserveAspectRatio="%s meet" viewBox="0 0 %d %d"%s>`,
 		`xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"`,
 		version.Version,
 		alignment,
 		w, h,
 		dimensions,
 	)
+	if !opts.FitScreen {
+		fitScreenWrapper = ""
+	}
```

Comment:
I think this should be the same as `setDimensions` above now. setting dimensions disables the fit to screen behavior.

Reply:
I think we also want to rename the option to  `--no-fit` or `--sized` since we want the fit behavior as default now